drivers/serial/uart_nrfx_uart.c

```diff
@@ -14,9 +14,36 @@
 
 
 #ifdef CONFIG_UART_INTERRUPT_DRIVEN
-static uart_irq_callback_t m_irq_callback; /**< Callback function pointer */
+
+static uart_irq_callback_t irq_callback; /**< Callback function pointer */
+
+/* Variable used to override the state of the TXDRDY event in the initial state
+ * of the driver. Before any byte is sent.the event is not set and we want
+ * to use it as an indication if the transmitter is ready to accept a new byte.
```

Comment:
"Before ..." -> "This event is not set by the hardware until a first byte is sent, and we want ..."

Reply:
Only this one correction needed. I can live with the other changes. ;-)

Reply:
done